### PR TITLE
Subtask/as 1192 appeal pdf publish

### PR DESCRIPTION
**Pull request by Fenisay**
## Ticket Number

AS-1192
AS-1450
AS-1451
AS-1453
AS-1454
AS-1456
AS-1457

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

**Review by rcmsoftwaredevelopment — APPROVED** [review]

**Review by rcmsoftwaredevelopment — APPROVED** [review]

**Comment by Fenisay** [comment]
@MrSimonEmms  Does it still make sense to let the process attempt in the document now that we are not in a cron job ?

**Review by rcmsoftwaredevelopment — APPROVED** [review]

**Review by MrSimonEmms — CHANGES_REQUESTED** [review]
Can you add the `DOCUMENT_SERVICE_URL` to the `horizon-householder-appeal-publish` in:
 - `/functions.yml`
 - `/charts/functions/values.yaml`
 - `/releases/dev/functions.yml`
 - `/releases/preprod/functions.yml`

I'm happy to do this as part of this PR if you would prefer

**Comment by MrSimonEmms** [comment]
> @MrSimonEmms Does it still make sense to let the process attempt in the document now that we are not in a cron job ?

I'm happy keeping it in there. I can envisage world where we try to upload immediately and then retry if a problem in future. One would hope that blob storage is reliable enough to never need to do that, but it doesn't affect functionality by being in there and never used

File: packages/forms-web-app/src/services/pdf.service.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ const storePdfAppeal = async (appeal) => {
     const msg = 'Error during the appeal pdf generation';
     log.error({ err }, msg);
 
-    throw new Error('Error during the appeal pdf generation');
+    throw new Error(msg);
```
[the remark this answers is not in the document]
Comment (Fenisay) [reply packages/forms-web-app/src/services/pdf.service.js:72]: Probably was your initial idea then someone pinged you on Slack

**Review by MrSimonEmms — APPROVED** [review]

**Pull request merged**